### PR TITLE
Adding flatbuffers

### PR DESCRIPTION
**Pull request by ct2034**
Please add the following dependency to the rosdep database.

## Package name:

flatbuffers

## Package Upstream Source:

https://github.com/google/flatbuffers

## Purpose of using this:

it is needed to decode the fbl logfiles of https://github.com/BehaviorTree/BehaviorTree.CPP

Distro packaging links:

## Links to Distribution Packages
- Ubuntu 
from jammy: https://packages.ubuntu.com/jammy/python3-flatbuffers
- Debian
In debian 11 there is https://packages.debian.org/bullseye/python3-flatbuffers
- Opensuse
https://software.opensuse.org/package/python-flatbuffers
- Redhat
https://mirrors.lug.mtu.edu/epel/9/Everything/x86_64/Packages/p/python3-flatbuffers-2.0.8-2.el9.noarch.rpm

There are no other distro packages available, so I propose to use https://pypi.org/project/flatbuffers/

### CODE REVIEW

**Review by tfoote — DISMISSED** [review]
It looks like your last commit got a lot of extra content into it (~1300 extra lines from a merge potentially) Please clean it up for us to be able to review.

**Comment by ct2034** [comment]
Sorry for that, @tfoote. My yaml formatter must have a different idea about yaml formatting ;)

**Review by adityapande-1995 — CHANGES_REQUESTED** [review]
Could you also add the key ``python3-flatbuffers`` for opensuse and rhel ?
https://software.opensuse.org/package/python-flatbuffers
https://mirrors.lug.mtu.edu/epel/9/Everything/x86_64/Packages/p/python3-flatbuffers-2.0.8-2.el9.noarch.rpm

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5883,6 +5883,30 @@ python3-flask-sqlalchemy:
...
+    '*': [python3-flatbuffers]
+    '8': null
+  ubuntu:
+    '*': [python3-flatbuffers]
+    focal:
+      pip:
+        packages: [flatbuffers]
+    bionic:
+      pip:
+        packages: [flatbuffers]
```
Comment (clalancette) [inline rosdep/python.yaml:5909]: This should be before `focal` so they are in alphabetical order.

**Review by clalancette — DISMISSED** [review]

**Review by mjcarroll — APPROVED** [review]

**Review by mjcarroll — APPROVED** [review]

**Review by adityapande-1995 — APPROVED** [review]

**Pull request merged**